.cbmc-batch/source/make_common_data_structures.c

```diff
@@ -21,6 +21,15 @@
 #include <evp_utils.h>
 #include <make_common_data_structures.h>
 
+void ensure_alg_properties_has_allocated_names(struct aws_cryptosdk_alg_properties *const alg_props) {
+    size_t md_name_size;
+    alg_props->md_name = can_fail_malloc(md_name_size);
```

Comment:
this is actually testing somewhat more cases than the function name suggests.  You either allocate, or have null.